### PR TITLE
Add event creator worker

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

File: src/github.com/matrix-org/dendron/dendron/main.go [hunk 1]
```diff
@@ -52,8 +52,10 @@ var (
 	federationSenderConfig = flag.String("federation-sender-config", "", "Federation sender worker config")
 	userDirectroyConfig    = flag.String("user-directory-config", "", "User directory worker config")
 	userDirectoryURLStr    = flag.String("user-directory-url", "", "The HTTP URL that the user directory will listen on")
-	frontendProxyConfig    = flag.String("frontend-proxy-config", "", "User directory worker config")
-	frontendProxyURLStr    = flag.String("frontend-proxy-url", "", "The HTTP URL that the user directory will listen on")
+	frontendProxyConfig    = flag.String("frontend-proxy-config", "", "Frontend proxy worker config")
+	frontendProxyURLStr    = flag.String("frontend-proxy-url", "", "The HTTP URL that the frontend proxy will listen on")
+	eventCreatorConfig    = flag.String("event-creator-config", "", "Event creator worker config")
```
Comment (richvdh) [inline src/github.com/matrix-org/dendron/dendron/main.go]: looks like this needs `gofmt`ing to align things

**Review by richvdh — APPROVED** [review]
looks plausible otherwise

**Comment by richvdh** [comment]
kinda wish we'd do this in sytest rather than dendron.

I assume we don't care about the jenkins failure - shall we vape the build config?

**Comment by erikjohnston** [comment]
> I assume we don't care about the jenkins failure - shall we vape the build config?

I had a look, and it fails with some perl error that I don't understand. Happy to vape the config given this project really only is to support sytest.

**Pull request merged**